src/GraphQLParser/AST/IHasDescription.cs

```diff
@@ -0,0 +1,7 @@
+namespace GraphQLParser.AST
+{
+    public interface IHasDescription
```

Comment:
Well, better to name as `IHasDescriptionNode` to align with `IHasDirectivesNode`.

Reply:
Yes, or vice versa.